lib/awestruct/extensions/disqus.rb

```diff
@@ -24,7 +24,7 @@ def disqus_comments()
             var disqus_identifier = #{identifier};
             (function() {
               var dsq = document.createElement("script"); dsq.type = "text/javascript"; dsq.async = true;
-              dsq.src = "http://#{site.disqus}.disqus.com/embed.js";
+              dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
```

Comment:
Where is `disqus_shortname` coming from? Shouldn't it be `site.disqus_shortname`?